crates/rolldown/tests/fixtures/tree_shaking/multi-declarator/artifacts.snap

```diff
@@ -0,0 +1,51 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/fixtures/tree_shaking/multi-declarator
+---
+# Assets
+
+## main.mjs
+
+```js
+import { default as assert } from "node:assert";
+
+//#region foo.js
+const foo = 'foo';
+const [c, d] = [1, 2];
```

Comment:
Both `c` and `d` is kept align with **esbuild**, 
Maybe we could support more advanced tree shaking in the future.

Reply:
https://esbuild.github.io/try/#YgAwLjIwLjIALS1idW5kbGUgLS1taW5pZnk9ZmFsc2UgLS1mb3JtYXQ9ZXNtAGUAZW50cnkuanMACmltcG9ydCB7IGZvbywgYyAgfSBmcm9tICcuL2ZvbycKCgphc3NlcnQuZXF1YWwoZm9vLCAnZm9vJykKYXNzZXJ0LmVxdWFsKGMsIDEpCgAAZm9vLmpzAGV4cG9ydCBjb25zdCBmb28gPSAnZm9vJywgYmFyID0gJ2JhcicKZXhwb3J0IGNvbnN0IFtjLCBkXSA9IFsxLCAyXQo